internal/httphandler/ledger_request_handler.go

```diff
@@ -33,6 +33,8 @@ func NewLedgerRequestHandler(db bcdb.DB, logger *logger.SugarLogger) http.Handle
 		logger:      logger,
 	}
 
+	// HTTP GET "/ledger/block/{blockId}?augmented=true" gets augmented block header
+	handler.router.HandleFunc(constants.GetBlockHeader, handler.blockQuery).Methods(http.MethodGet).Queries("augmented", "{isAugmented:true|false}")
 	// HTTP GET "/ledger/block/{blockId}" gets block header
 	handler.router.HandleFunc(constants.GetBlockHeader, handler.blockQuery).Methods(http.MethodGet)
```

Comment:
I think, we should use a single REST API endpoint. When setting `isAugmented=false`, we get the old behaviour.

Reply:
or when when this key isAugmented is missing, resort to old behavior

Reply:
This is exactly how it works. Handler route in line 37 just indicated that we want to look for query parameter `augmented`.  Leaving only route in line 39 ignores all query parameters...